src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -0,0 +1,105 @@
+import './NewCommentForm.scss';
+import React, { FormEvent, useState } from 'react';
+import { Comment } from '../../types';
+
+type Props = {
+  commentsList: Comment[] | undefined;
+  setCommentsList: (arg:Comment[]) => void;
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+                                                  commentsList, setCommentsList,
+                                                }) => {
+  const [yourname, setYourname] = useState('');
+  const [youremail, setYouremail] = useState('');
+  const [yourcomment, setYourcomment] = useState('');
```

Comment:
Use camel case for more readability

```suggestion
  const [yourName, setYourName] = useState('');
  const [yourEmail, setYourEmail] = useState('');
  const [yourComment, setYourComment] = useState('');
```